deployment/common/changeset/internal/ops/ops_mcm_deploy.go

```diff
@@ -0,0 +1,90 @@
+package ops
+
+import (
+	"github.com/Masterminds/semver/v3"
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+
+	"github.com/ethereum/go-ethereum/core/types"
+	bindings "github.com/smartcontractkit/ccip-owner-contracts/pkg/gethwrappers"
+
+	cldf_evm "github.com/smartcontractkit/chainlink-deployments-framework/chain/evm"
+	cldf "github.com/smartcontractkit/chainlink-deployments-framework/deployment"
+
+	"github.com/smartcontractkit/chainlink-deployments-framework/operations"
+	mcmsnew_zksync "github.com/smartcontractkit/chainlink/deployment/common/changeset/internal/evm/zksync"
+)
+
+type OpEVMMCMSDeps struct {
```

Comment:
if you're onboard with renaming to `mcms` to `mcm`, I think we should do it across the board.
```suggestion
type OpEVMMCMDeps struct {
```

Reply:
This alone is a common dependency for few other MCMS Operations, hence will leave it as such